### PR TITLE
Fix data driven manager

### PR DESCRIPTION
**Pull request by DanielRGonzalesV**
Fix and Impromevent data driven manager

### CODE REVIEW

File: src/main/java/com/salesforce/dev/pages/objects/CampaignGenie.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import com.sforce.soap.partner.PartnerConnection;
+import com.sforce.soap.partner.sobject.SObject;
+import com.sforce.ws.ConnectionException;
+import org.apache.log4j.Logger;
+
+/**
+ * Created by Veronica Prado on 8/25/2015.
+ * Class to get data related to Campaign
+ */
+public class CampaignGenie {
```
Comment (carledriss) [inline src/main/java/com/salesforce/dev/pages/objects/CampaignGenie.java:20]: Is this class necessary?

File: src/main/java/com/salesforce/dev/pages/objects/ContactGenie.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import com.sforce.soap.partner.sobject.SObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by alex on 06/09/2015.
+ */
+public class ContactGenie {
```
Comment (carledriss) [inline src/main/java/com/salesforce/dev/pages/objects/ContactGenie.java:16]: Is this class necessary?

File: src/test/java/com/salesforce/dev/chatter/EditPostChatter.java [hunk 3]
```diff
@@ -47,8 +46,8 @@ public void CreatePostAndComment() {
     }
 
     private Chatter getChatter(String fileJson) {
-        Iterator<Chatter[]> chattersData = dataDrivenManager.getChatter(fileJson);
-        return chattersData.next()[0];
+        Iterator<Object[]> chattersData = DataDrivenManager.getObjects(fileJson, Chatter.class);
+        return (Chatter) chattersData.next()[0];
```
Comment (carledriss) [inline src/test/java/com/salesforce/dev/chatter/EditPostChatter.java:50]: Did you try to use the JSONMapper here?
Review it with @MijhailVillarroel .

**Comment by DanielRGonzalesV** [comment]
ok.. I will review it

**Pull request closed without merging**